.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,43 @@
+## Description
+<!--
+Thank you so much for your PR!  To help us review your contribution, please
+consider the following points:
+
+- The PR title should summarize the changes, for example "Add new group argument for the
+  pair plot".  Avoid non-descriptive titles such as "Addresses issue #348". If your pull 
+  request addresses an issue, please use the pull request title to describe 
+  the issue and mention the issue number in the pull request description.
+
+- The description should provide at least 1-2 sentences describing the pull request
+  in detail (Why is this change required?  What problem does it solve?) and
+  link to any relevant issues. If modifying a plot, render your plot to inspect for changes 
+  and copy image in the pull request message on Github
+
+- Please prefix the title of incomplete contributions with [WIP] (to indicate a work in 
+  progress). WIPs may be useful to (1) indicate you are working on something to avoid 
+  duplicated work, (2) request broad review of functionality or API, or (3) seek collaborators.
+-->
+<!-- Briefly describe the bug you are trying to fix -->
+
+<!-- Describe your changes in detail -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Is this PR following the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
```

Comment:
I think it would be "does this PR follow" but please check, English is not my first language.

Reply:
Yes, it should be 'does this PR follow' in this case, to follow the same pattern( we are using 'Has added new tests' etc). Thanks for the fix :)